core/src/com/unciv/ui/utils/Log.kt

```diff
@@ -0,0 +1,209 @@
+package com.unciv.utils
+
+import java.time.Instant
+import java.util.regex.Pattern
+
+
+object Log {
+
+    /** Add -DnoLog=<comma-separated-list-of-class-names> to not log these classes. */
+    private val disabledLogsFromProperty = System.getProperty("noLog")?.split(',')?.toMutableSet() ?: mutableSetOf()
+
+    /** Log tags (= class names) **containing** the following Strings will not be logged.  */
+    val disableLogsFrom = if (disabledLogsFromProperty.isEmpty()) {
```

Comment:
Who mutates this? Us when debugging?
If system properties is the main way to choose, I would prefer a more flexible design - you do either system props or defaults - I'd let props switch defaults individually.. Just an idea.
À propos - some dox for the dev "I want a run with Music logged: howTo" might be good to have, even right here in this file I'd find them. Plus wiki probably.

Reply:
I just meant one might wonder as to how one is 'supposed to' do this. Like "can one do run configs for this"?... You know, some people go looking for rules even if they see a viable way already. Might be the rules will work longer than the first shortcut...